### PR TITLE
Release 1.1.1 to conda-forge

### PR DESCRIPTION
**Pull request by goel-akas**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by goel-akas** [comment]
NOTE: We will revert this commit and set the latest to 1.3.2 again, once 1.1.1 is available on conda-forge.

**Comment by goel-akas** [comment]
conda smithy render:
```
(base) ➜  sagemaker-code-editor-feedstock git:(main) conda smithy rerender -c auto
INFO:conda_smithy.configure_feedstock:Downloading conda-forge-pinning-2024.10.02.19.54.49
INFO:conda_smithy.configure_feedstock:Extracting conda-forge-pinning to /Users/goelakas/.cache/conda-smithy
INFO:conda_smithy.configure_feedstock:__pycache__ rendering is skipped
INFO:conda_smithy.configure_feedstock:README rendering is skipped
WARNING: Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
WARNING: Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
WARNING: No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.26
Adding in variants from internal_defaults
Adding in variants from /Users/goelakas/.cache/conda-smithy/conda_build_config.yaml
Adding in variants from /Users/goelakas/git-workspace/sagemaker-code-editor-feedstock/recipe/conda_build_config.yaml
Adding in variants from argument_variants
subpackages_about [('sagemaker-code-editor', {'home': 'https://aws.amazon.com/sagemaker/', 'license': 'MIT', 'license_family': 'MIT', 'license_file': ['LICENSE', 'LICENSE.vscode.txt', 'LICENSE-THIRD-PARTY', 'ThirdPartyNotices.vscode.txt'], 'summary': 'Code Editor based on Code-OSS, Visual Studio Code Open Source'})]
INFO:conda_smithy.configure_feedstock:Re-rendered with conda-build 24.9.0, conda-smithy 3.41.1, and conda-forge-pinning 2024.10.02.19.54.49
[main ea199d7] MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.41.1, and conda-forge-pinning 2024.10.02.19.54.49
 5 files changed, 10 insertions(+), 4 deletions(-)
INFO:conda_smithy.configure_feedstock:
INFO:conda_smithy.configure_feedstock:These changes need to be pushed to github!
```

File: recipe/meta.yaml
Comment (aws-prayags) [inline recipe/meta.yaml:10]: are we not using different branches for each minor version? Managing multiple versions using one branch will be error prone

**Review by aws-pangestu — APPROVED** [review]

File: recipe/meta.yaml
Comment (aws-pangestu) [reply recipe/meta.yaml:10]: Akash is planning to revert this change once the build has deployed to condaforge. Definitely a good idea to start a 1.1.1 branch from this commit though, once it has been merged

Comment (goel-akas) [reply recipe/meta.yaml:10]: I agree, this is not preferable. One question was whether this release process works with a non-main branch. We can try it with this change (I can open a new PR).

Comment (aws-prayags) [reply recipe/meta.yaml:10]: from reading [this section](https://conda-forge.org/docs/maintainer/updating_pkgs/#forking-and-pull-requests), it does look like any branch would work

yes, let's try it out

**Pull request closed without merging**
